cli/internal/ci/ci.go

```diff
@@ -18,6 +18,7 @@ func Name() string {
 
 // Info returns information about a CI vendor
 func Info() Vendor {
+	// key=val required
```

Comment:
What does this comment mean? Maybe flesh it out

Reply:
Ah good call - will do.